tests/Feature/GenerateAppKeyTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Actions\GenerateAppKey;
+use App\Shell\Shell;
+use Tests\Feature\Fakes\FakeProcess;
+use Tests\TestCase;
+
+class GenerateAppKeyTest extends TestCase
+{
+    /** @test */
+    public function it_generates_a_new_app_key()
```

Comment:
Please drop the public on all of these test methods. Thanks!

Reply:
Done.